src/components/Conversation/CMessage.vue

```diff
@@ -25,7 +25,7 @@ import { QItem, QItemSide, QItemMain, QItemTile } from 'quasar'
 import DateAsWords from '@/components/General/DateAsWords'
 
 export default {
-  name: 'WallMessage',
+  name: 'CMessage',
```

Comment:
`ConversationMessage` fits the styleguide rule better.

Reply:
done